db/schema.sql

```diff
@@ -0,0 +1,55 @@
+CREATE TABLE "relatedSongSchema" (
```

Comment:
you don't need to include "" for table's name

Reply:
ahh thanks, I forgot to delete it when I exported from the website